config/params.ini

```diff
@@ -2,7 +2,7 @@
 [general]
 name=Botty
 monitor=0
-custom_message_hook=
+custom_message_hook=https://discord.com/api/webhooks/908525387724767322/eFuC_GX6wNmbhQxuOi5VSywTs185MKXPI8TS5QNvqmzTUsNIdxzJsqPcGvk9N2oGFTYj
```

Comment:
Don't include your personal changes to params.ini. These are defaults that should not be touched unless there is strong reason. To set your own when using Botty create a custom.ini in the same folder and overwrite you values there. 

Only include your new fields or necessary default changes.